@vates/parse-duration/index.js

```diff
@@ -6,6 +6,9 @@ exports.parseDuration = value => {
   if (typeof value === 'number') {
     return value
   }
+  if (typeof value !== 'string' || value === '') {
+    throw new TypeError(`not a valid duration: ${value}`)
+  }
```

Comment:
You can rearrange the code to avoid duplicating the error:

```js
if (typeof value === 'string' && value !== '') {
  const duration = ms(value)
  if (duration !== undefined) {
    return duration
  }
}
throw new TypeError(`not a valid duration: ${value}`)
```